File: 24.07.2024/calculator.py
def calculator():
    operator = input("Enter an operator (+, -, *, /): ")
    first = float(input("Enter first number: "))
    second = float(input("Enter second number: "))
    if operator == '+':
        print("Result:", first + second)
    elif operator == '-':
        print("Result:", first - second)
    elif operator == '*':
        print("Result:", first * second)
    elif operator == '/':
        if second != 0:
            print("Result:", first / second)
        else:
            print("Error! Division by zero.")
    else:
        print("Invalid operator")
calculator()

File: 24.07.2024/calculator.c
#include <stdio.h>
int main() {
    char operator;
    double first, second;
    printf("Enter an operator (+, -, *, /): ");
    scanf("%c", &operator);
    printf("Enter two operands: ");
    scanf("%lf %lf", &first, &second);

    switch (operator) {
        case '+':
            printf("%.2lf + %.2lf = %.2lf\n", first, second, first + second);
            break;
        case '-':
            printf("%.2lf - %.2lf = %.2lf\n", first, second, first - second);
            break;
        case '*':
            printf("%.2lf * %.2lf = %.2lf\n", first, second, first * second);
            break;
        case '/':
            if (second != 0.0)
                printf("%.2lf / %.2lf = %.2lf\n", first, second, first / second);
            else
                printf("Division by zero error\n");
            break;
        default:
            printf("Invalid operator\n");
    }
    return 0;
}
